test/JDBC/expected/babel_delete.out

```diff
@@ -207,7 +205,9 @@ go
 
 ~~ROW COUNT: 3~~
 
-~~ROW COUNT: 1~~
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: ctid is NULL)~~
```

Comment:
This is a regression that will happen in some particular outer join cases, haven't got a full picture of the reason or pattern yet. Will file a Jira for this as bug fix.